### DISCUSSION TITLE
Custom Diffusers Pipeline

### DISCUSSION THREAD

**Discussion by setothegreat** [vladmandic/sdnext#2600] [Q&A]
I just noticed that support for custom diffusers pipelines was added a bit more than a month ago (huzzah!)
This has been a feature I've been looking forward to for nearly a year now. However, I am having a bit of trouble finding documentation on how to go about setting it up in the settings, as there is no tool tip and I cannot find any discussion of it on this GitHub.

[Using Long Prompt Weighting XL as an example](https://github.com/huggingface/diffusers/blob/main/examples/community/lpw_stable_diffusion_xl.py), would I essentially just download the .py file to a given location, and then enter the path to that file into the text box in the settings menu and then restart the server? Or is there anything else that needs to be done?

Thanks a ton for adding this feature!

**Comment by vladmandic** [comment]
settings -> diffusers -> pipeline: custom
settings -> diffusers -> load custom: name of a custom pipeline as seen in diffusers repo

do not download py files, what's needed is going to be downloaded and used automatically.  
i plan to expand on this feature in the future to make it more user-friendly.
